src/pages/wallet/readonly/add.js

```diff
@@ -135,16 +135,20 @@ class AddReadOnlyWallet extends Component {
         }
       }
       if (newAddress.startsWith('0x')) {
-        chain = 'Rootstock';
+        chain = Chain.Rootstock;
         symbol = 'RBTC';
       } else {
-        chain = 'Bitcoin';
+        chain = Chain.Bitcoin;
         symbol = 'BTC';
       }
```

Comment:
Every address object in database already has a "chain" property. You should be able to get it from address.get("chain").